demos/tests/cases.py

```diff
@@ -460,6 +460,27 @@ def single_option_cases(key, *args):
                 '--output_names': 'output_s,output_e',
                 '--vocab': str(OMZ_DIR / 'models/intel/bert-small-uncased-whole-word-masking-squad-0002/vocab.txt'),
             }),
+            TestCase(options={
+                '-m': ModelArg('bert-small-uncased-whole-word-masking-squad-int8-0002', precision='FP32-INT8'),
+                '--input_names': 'input_ids,attention_mask,token_type_ids,position_ids',
+                '--output_names': 'output_s,output_e',
+                '--vocab':
+                    str(OMZ_DIR / 'models/intel/bert-small-uncased-whole-word-masking-squad-int8-0002/vocab.txt'),
+            }),
+            TestCase(options={
+                '-m': ModelArg('bert-large-uncased-whole-word-masking-squad-0001'),
+                '--input_names': '0,1,2',
+                '--output_names': '3171,3172',
+                '--vocab': str(
+                    OMZ_DIR / 'models/intel/bert-large-uncased-whole-word-masking-squad-0001/vocab.txt'),
```

Comment:
The formatting here is inconsistent with the other networks.

Reply:
Updated, thanks!